react-js/react.md

```diff
@@ -4,14 +4,20 @@
 
 8 Hours
```

Comment:
Can you update this total. It no longer matches your subtotals.

Reply:
Done